packages/react-components/src/components/table-output-component.tsx

```diff
@@ -62,11 +63,25 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 onGridReady={this.onGridReady}
                 components={this.components}
                 enableColResize={true}
+                onCellClicked={this.onEventClick}
+                rowSelection='single'
             >
             </AgGridReact>
         </div>;
     }
 
+    private onEventClick(event: CellClickedEvent) {
+        const columns = event.columnApi.getAllColumns();
+        const timeStampHeader = columns.find(column => column.getColDef().headerName === "Timestamp (ns)");
```

Comment:
The column header string is acutally "Timestamp ns" in the trace compass server.

Reply:
If we need this, then it's a sign there's a problem with the TSP, this is the kind of string that should never be hard-coded. Time to work on https://github.com/theia-ide/trace-server-protocol/issues/35?